src/sentry/static/sentry/app/components/panels/index.jsx

```diff
@@ -0,0 +1,8 @@
+import Panel from './panel';
+import PanelHeader from './panelHeader';
+import PanelBody from './panelBody';
+import PanelFooter from './panelFooter';
+import PanelItem from './panelItem';
+import PanelAlert from './panelAlert';
+
+export {Panel, PanelHeader, PanelBody, PanelFooter, PanelItem, PanelAlert};
```

Comment:
What are your thoughts on named vs default exports?

I think we should stick to default export since that's what most of our modules are.

What about sticking the components on to `Panel` so that there is only only thing to import?

Reply:
I think we should do both, but I prefer using named exports and I think it makes sense since it's a collection of components, where most of our other modules are a single one